website/src/pages/DailyPage/DailyPage.jsx

```diff
@@ -260,18 +245,14 @@ const Daily = () => {
                 />
               );
             })}
-          </div>
-          <figure className="line"></figure>
-          {dailyTextLoading ? (
-            <div className="loadingSpinner">
-              <RingLoader
-                loading={dailyTextLoading}
-                color="#E77002"
-                size={300}
-              />
-            </div>
-          ) : benchmarkType !== "" ? (
-            <div className="daily__container">
+          </section>
+
+          <figure className="p-page w-full">
+            <div className="border-front border" />
+          </figure>
+
+          {!dailyTextLoading && benchmarkType !== "" && (
+            <section className="p-page mt-12">
```

Comment:
When clicking on the benchmark type I am not seeing any graph, is that expected?

Reply:
<img width="922" alt="image" src="https://github.com/vitessio/arewefastyet/assets/35779988/b28d09a5-b405-452b-aa42-b87af0ded3e9">